### PR TITLE
Update README and SDK version

### PR DESCRIPTION
**Pull request by anitachen-appier**
### Purpose

1. Fix typo and format errors
2. Add Kotlin sample code

### Result

All the sample codes should work in real apps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,55 @@
...
+// Kotlin
+tracker.setVisitorProperty("name", "User Name")
+tracker.setVisitorProperty("email", "user@company.com")
+```
+
 Or,
 
 ``` java
+// Java
 Map<String,String> visitorProps = new HashMap<String,String>();
```
Comment (cedricshih) [inline README.md:34]: imo: `Map<String, Object> visitorProps = new HashMap<>();`?

File: README.md [hunk 2]
```diff
@@ -1,33 +1,55 @@
...
 visitorProps.put("name", "User Name");
 visitorProps.put("email", "user@company.com");
-tracker.setVisitorProperty(visitorProps);
+tracker.setVisitorProperties(visitorProps);
+```
+
+``` kotlin
+// Kotlin
+val visitorProps: MutableMap<String, String> = HashMap()
```
Comment (cedricshih) [inline README.md:42]: imo: `MutableMap<String, Any>`? (not very sure about `Any`)

**Review by cedricshih — APPROVED** [review]
static code approved

**Review by wubai-appier — APPROVED** [review]
Approved with static code review.

**Pull request merged**